test/clt-tests/integrations/kafka/test-integration-kafka-ms.rec

```diff
@@ -32,11 +37,35 @@ docker exec manticore stdbuf -oL searchd --logdebugvv > /dev/null 2>&1; echo $?
 ––– output –––
 0
 ––– input –––
-docker run -it -d --network=app-network --name kafka -v ./test/clt-tests/integrations/kafka/import.sh:/import.sh -v ./test/clt-tests/integrations/kafka/dump.json:/tmp/dump.json -e KAFKA_CFG_NODE_ID=0 -e KAFKA_CFG_PROCESS_ROLES=controller,broker -e KAFKA_CFG_CONTROLLER_QUORUM_VOTERS=0@kafka:9093 -e KAFKA_CFG_LISTENERS=PLAINTEXT://:9092,CONTROLLER://:9093 -e KAFKA_CFG_ADVERTISED_LISTENERS=PLAINTEXT://:9092 -e KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP=CONTROLLER:PLAINTEXT,PLAINTEXT:PLAINTEXT -e KAFKA_CFG_CONTROLLER_LISTENER_NAMES=CONTROLLER -e KAFKA_CFG_INTER_BROKER_LISTENER_NAME=PLAINTEXT bitnamilegacy/kafka:3.7.0 > /dev/null 2>&1; echo $?
+docker run -it -d --network=app-network --name kafka --platform linux/amd64 \
+-v ./test/clt-tests/integrations/kafka/import.sh:/import.sh \
+-v ./test/clt-tests/integrations/kafka/dump.json:/tmp/dump.json \
+-e KAFKA_NODE_ID=1 \
+-e KAFKA_PROCESS_ROLES=broker,controller \
+-e KAFKA_CONTROLLER_QUORUM_VOTERS=1@kafka:9093 \
+-e KAFKA_LISTENERS=PLAINTEXT://:9092,CONTROLLER://:9093 \
+-e KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://kafka:9092 \
+-e KAFKA_LISTENER_SECURITY_PROTOCOL_MAP=CONTROLLER:PLAINTEXT,PLAINTEXT:PLAINTEXT \
+-e KAFKA_CONTROLLER_LISTENER_NAMES=CONTROLLER \
+-e KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR=1 \
+-e KAFKA_GROUP_INITIAL_REBALANCE_DELAY_MS=0 \
+-e CLUSTER_ID=MkU3OEVBNTcwNTJENDM2Qk \
+apache/kafka:$KAFKA_VERSION > /dev/null 2>&1; echo $?
 ––– output –––
 0
 ––– input –––
-docker exec kafka kafka-topics.sh --create --topic my-data --partitions 4 --bootstrap-server localhost:9092 2>&1 | grep -o 'Created topic my-data\.' | head -n 1
+sleep 5
```

Comment:
Since we’re already checking below whether Kafka is available, this sleep isn’t needed